### PR TITLE
Add bio field to the Profile struct

### PR DESCRIPTION
**Pull request by DogukanGun**
Add bio field to the Profile struct . 
 This pull request was created for https://app.gib.work/bounties/0d295331-b491-4356-8bc5-046a618dcfd8 in an attempt to solve a bounty #51 . Payment for the bounty is immediately sent to the contributor after merge.

### CODE REVIEW

File: anchor/programs/pubkey-protocol/src/instructions/profile/profile_create.rs [hunk 1]
```diff
@@ -65,6 +65,7 @@ pub fn profile_create(ctx: Context<ProfileCreate>, args: ProfileCreateArgs) -> R
         username,
         name,
         avatar_url,
+        bio,
```
Comment (beeman) [inline anchor/programs/pubkey-protocol/src/instructions/profile/profile_create.rs]: We don't need the bio in the `profile_create` instruction, only in the `profile_update` one.

File: anchor/programs/pubkey-protocol/src/state/profile.rs [hunk 2]
```diff
@@ -1,3 +1,5 @@
+use std::char::MAX;
```
Comment (beeman) [inline anchor/programs/pubkey-protocol/src/state/profile.rs]: This one seems to be obsolete.

File: cli/src/commands/provision-sample-data-profiles.ts [hunk 3]
```diff
@@ -127,9 +130,9 @@ export async function provisionSampleDataProfiles(options: ProvisionSampleDataOp
 
   const profilesIdentitiesAdd = Object.keys(profileProvisionMap)
     .filter((username) => Object.keys(profileProvisionMap[username].identities ?? []).length > 0)
-    .map((username) => ({ identities: profileProvisionMap[username].identities, username }))
+    .map((username) => ({ identities: profileProvisionMap[username].identities, username, bio:profileProvisionMap[username].create.bio }))
```
Comment (beeman) [inline cli/src/commands/provision-sample-data-profiles.ts]: Profile bio is not a part of the `identity`

File: sdk/src/lib/pubkey-protocol-sdk-interfaces.ts [hunk 4]
```diff
@@ -96,6 +97,7 @@ export interface ProfileIdentityAddOptions {
   provider: IdentityProvider
   providerId: string
   username: string
+  bio?: string
```
Comment (beeman) [inline sdk/src/lib/pubkey-protocol-sdk-interfaces.ts]: Profile bio is not a part of the `identity`

**Review by beeman — CHANGES_REQUESTED** [review]
Hey @DogukanGun - left some comments on this pr!

**Comment by DogukanGun** [comment]
Hi @beeman , thanks for the review. I updated the code according to your comments.

**Review by beeman — APPROVED** [review]

**Comment by beeman** [comment]
Thanks a lot for your second PR @DogukanGun!

**Pull request merged**